### PR TITLE
adding information about the branches in NorESMhub/CTSM repository

### PR DESCRIPTION
**Pull request by huitang-earth**
### Description of changes

I have added the information about the branches in NorESMhub/CTSM repository. 

This is just a testing PR. Please check if the information is correct. I will send another PR after tomorrow's discussion.

### Specific notes

Contributors other than yourself, if any:
No

CTSM Issues Fixed (include github issue #):
#5 

Are answers expected to change (and if so in what way)?
No

Any User Interface Changes (namelist or namelist defaults changes)?
No

Testing performed, if any:
(List what testing you did to show your changes worked as expected)
(This can be manual testing or running of the different test suites)
(Documentation on system testing is here: https://github.com/ESCOMP/ctsm/wiki/System-Testing-Guide)
(aux_clm on cheyenne for intel/gnu and izumi for intel/gnu/nag/pgi is the standard for tags on master)
No

**NOTE: Be sure to check your Coding style against the standard:**
https://github.com/ESCOMP/ctsm/wiki/CTSM-coding-guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Branches in NorESMhub/CTSM
+
+This repository has the following CTSM branches working with NorESM2:
+
+- *master* - keep track with the master branch of [ESCOMP/CTSM](https://github.com/ESCOMP/CTSM).
+  - This branch is used by [NorESMhub/NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform)
+  - This branch can be used for running the most recent version of [FATES](https://github.com/NGEET/fates)
+
+- *release-clm5.0.14-Nor* - The working branch used by NorESM2. Currently, all the pull requests should be sent to this branch.
```
Comment (kjetilaas) [inline README.md]: I would suggest specifying that only PRs that are not answer changing should be sent to this branch. New developments should be sent elsewhere. Currently there is a clm-nor-dev, which should be used for ongoing developments, but I would also suggest renaming this.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Branches in NorESMhub/CTSM
+
+This repository has the following CTSM branches working with NorESM2:
+
+- **master** - keep track with the master branch of [ESCOMP/CTSM](https://github.com/ESCOMP/CTSM).
+  - This branch is used by [NorESMhub/NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform)
+  - This branch can be used for running the most recent version of [FATES](https://github.com/NGEET/fates)
+
+- **release-clm5.0.14-Nor** - The working branch used by NorESM2. Currently, all the pull requests should be sent to this branch.
```
Comment (MichaelSchulzMETNO) [inline README.md:9]:
```suggestion
- **release-clm5.0.14-Nor** - The working branch used by NorESM2. Non-answer changing (with respect to NorESM2s CMIP6 version) pull requests should be sent to this branch. For other changes, new branches discuss with CTSM and NorESM core teams.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Branches in NorESMhub/CTSM
+
+This repository has the following CTSM branches working with NorESM2:
```
Comment (MichaelSchulzMETNO) [inline README.md:3]:
```suggestion
This repository has the following CTSM branches intended to facilitate working with NorESM2:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+- **master** - keep track with the master branch of [ESCOMP/CTSM](https://github.com/ESCOMP/CTSM).
+  - This branch is used by [NorESMhub/NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform)
+  - This branch can be used for running the most recent version of [FATES](https://github.com/NGEET/fates)
+
+- **release-clm5.0.14-Nor** - The working branch used by NorESM2. Currently, all the pull requests should be sent to this branch. 
+  - Tags being developed from this branch are:
```
Comment (MichaelSchulzMETNO) [inline README.md:10]:
```suggestion
  - Tags in this branch help identifying the link to the NorESM2 releases:
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    - release-clm5.0.14-Nor_v1.0.2 - the tag working with NorESM2 master branch and tags: release-noresm2.0.4, release-noresm2.0.3
```
Comment (MichaelSchulzMETNO) [inline README.md:11]:
```suggestion
    - release-clm5.0.14-Nor_v1.0.2 - the tag working with NorESM2 noresm2 tags: release-noresm2.0.4, release-noresm2.0.3
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    - release-clm5.0.14-Nor_v1.0.1 - the tag working with NorESM2 tags: release-noresm2.0.2 
+    - release-clm5.0.14-Nor_v1.0.0 - the tag working with NorESM2 tags: release-noresm2.0.1, release-noresm2.0.0 
+
+- **release-cesm2.2.01-Nor** - The branch used for the next phase of NorESM2 (under development).
```
Comment (MichaelSchulzMETNO) [inline README.md:15]:
```suggestion
- **release-cesm2.2** - Copy of ESCOMP branch, base for release-cesm2.2.01-Nor branch.
- **release-cesm2.2.01-Nor** - The branch used for the next phase of NorESM2 (under development).
```

**Review by MichaelSchulzMETNO — CHANGES_REQUESTED** [review]
Please accept my small suggested edits

**Pull request merged**